internal/tiltfile/local_resource.go

```diff
@@ -34,12 +35,13 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 
 	if err := s.unpackArgs(fn.Name(), args, kwargs,
 		"name", &name,
-		"cmd", &cmd,
+		"cmd?", &updateCmdStr,
 		"deps?", &deps,
 		"trigger_mode?", &triggerMode,
 		"resource_deps?", &resourceDepsVal,
 		"ignore?", &ignoresVal,
 		"auto_init?", &autoInit,
+		"serveCmd?", &serveCmdStr,
```

Comment:
all args should be slug-cased, so this should be `serve_cmd`

Reply:
(if you end up deciding to keep it as-is, ok, i'll stop commenting now!)

Reply:
Nope this is great, thank you!